diagnostic.md

```diff
@@ -45,6 +62,9 @@ weather === "sunny"
 
 What do these expressions evaluate to?  Explain your answers.
 
+weather = "sunny" --> evaluates to 'sunny'. Weather is now assigned a string value of "sunny".
+
+weather === "sunny" --> the strict equality (===) evaluates to true. This happened because "sunny" is a truthy value.
```

Comment:
Good. You just forgot the first line of `let weather` evaluates to undefined. Got the rest though!